lib/pages/chat_list/chat_list.dart

```diff
@@ -486,7 +486,7 @@ class ChatListController extends State<ChatList>
         );
         if (await client.encryption?.keyManager.isCached() == false ||
             await client.encryption?.crossSigning.isCached() == false ||
-            client.isUnknownSession && !mounted) {
+            client.isUnknownSession && mounted) {
```

Comment:
why you need to change it ?

Reply:
we can not show the dialog if the ChatList is not mounted